File: _examples/figure-table/_code-snippets/big-penguins.R
penguins |>
  arrange(desc(body_mass_g)) |>
  slice_head(n = 5) |>
  select(species, sex, island, body_mass_g, flipper_length_mm) |>
  gt() |>
  cols_label(
    body_mass_g = "Body Mass (g)",
    flipper_length_mm = "Flipper Length (mm)"
  )

File: _examples/figure-table/_code-snippets/big-penguins.py
(pl.DataFrame(penguins)
 .sort("body_mass_g", descending=True)
 .head(5)
 .select(["species", "sex", "island", "body_mass_g", "flipper_length_mm"])
 .pipe(GT)
 .cols_label(
     body_mass_g="Body Mass (g)",
     flipper_length_mm="Flipper Length (mm)"
 )
)